SerratusApi/Controllers/FamilySectionsController.cs

```diff
@@ -38,12 +38,18 @@ public async Task<ActionResult<IEnumerable<FamilySection>>> GetFamilySections()
         }
 
         [HttpGet("get-runs/{family}")]
-        public async Task<ActionResult<IEnumerable<FamilySection>>> GetRunsFromFamily(string family)
+        public async Task<ActionResult<IEnumerable<FamilySection>>> GetRunsFromFamily(string family, [FromQuery] int page)
         {
+            var recordsPerPage = 10;
+            if (page == 0)
```

Comment:
I'm guessing this is for the case when `page` is not specified. can we specify a parameter default in the function header to be more concise?

```cs
GetRunsFromFamily(string family, [FromQuery] int page = 1)
```

relevant doc: https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/named-and-optional-arguments#optional-arguments

Reply:
That was already in there V!

Reply:
I forgot I had already been sending "1" from the front end as the first page anyway, good catch